src/applications/travel-pay/components/AppointmentsTable.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const CLAIM_STATUS = {
+  SAVED: 'Saved',
+  INCOMPLETE: 'Incomplete',
+  IN_PROCESS: 'In Process',
+  CLAIM_SUBMITTED: 'Claim Submitted',
+  MANUAL_REVIEW: 'In Manual Review',
+  ON_HOLD: 'On Hold',
+  APPEALED: 'Appealed',
+  // Closed as a bunch of variants.
+  // TBD if we need to be more specific.
+  CLOSED: 'Closed',
+};
+
+export default function AppointmentsTable() {
+  return (
+    <va-table table-title="Travel claims">
+      <va-table-row slot="headers">
+        <span>Travel claim Number</span>
+        <span>Status</span>
+        <span>Details</span>
+      </va-table-row>
+      <va-table-row>
+        <span>#1234567</span>
+        <span>
+          <StatusPill status={CLAIM_STATUS.IN_PROCESS} />
+        </span>
+        <span>Consequat cillum, voluptate labore.</span>
+      </va-table-row>
+      <va-table-row>
+        <span>#09876543</span>
+        <span>
+          <StatusPill status={CLAIM_STATUS.MANUAL_REVIEW} />
+        </span>
+        <span>---</span>
+      </va-table-row>
+    </va-table>
```

Comment:
This is cool. Did you also take a look at how the Claims Status Tool does their list page?

I think @micahtaylor mentioned looking into the CST and I'm wondering if we could `synergize` there. The way the CST does it also lends itself well to diving into the details of a claim, should we need to do that in the future.

Reply:
removed this table and updated with the design that's similar to CST!